ReactAndroid/src/main/java/com/facebook/react/modules/network/NetworkingModule.java

```diff
@@ -311,11 +360,17 @@ private void readWithProgress(
       int requestId,
       ResponseBody responseBody) throws IOException {
     Reader reader = responseBody.charStream();
+    ProgressResponseBody progressResponseBody = (ProgressResponseBody)responseBody;
```

Comment:
@philikon we’re blocked on this line. We have an internal test where this is crashing with java.lang.ClassCastException: `okhttp3.internal.http.RealResponseBody cannot be cast to com.facebook.react.modules.network.ProgressResponseBody`. That test uses a different network implementation, that calls into `NetworkingModule.sendRequest`